modules/social_features/social_group/social_group.install

```diff
@@ -1025,3 +1025,18 @@ function social_group_update_11902(): void {
   }
 
 }
+
+/**
+ * Cancel limit of outputted roles in views.view.group_manage_members.
+ */
+function social_group_update_11903(): string {
```

Comment:
@VBaldych Since it will be released already in version 12.x, please change number to `12001`

Reply:
Done